lib/tesla/mock.ex

```diff
@@ -229,23 +229,12 @@ defmodule Tesla.Mock do
 
   # Gets the mock fun for the current process or its ancestors
   defp pdict_get do
-    pid_holder =
-      Enum.find(Process.get(:"$ancestors", []), self(), fn ancestor ->
-        is_holder? =
-          ancestor
-          |> Process.info()
-          |> Keyword.get(:dictionary)
-          |> Keyword.get(__MODULE__)
-
-        !is_nil(is_holder?)
-      end)
-      |> case do
```

Comment:
That `case` statement is a leftover from the previous solution that assumed using non-existent `Process.get(pid, value)`. Now, it is not necessary.